scripts/shared/clusters.sh

```diff
@@ -37,7 +55,7 @@ olm_version="${FLAGS_olm_version}"
 settings="${FLAGS_settings}"
 timeout="${FLAGS_timeout}"
 
-echo "Running with: k8s_version=${k8s_version}, olm_version=${olm_version}, olm=${olm}, globalnet=${globalnet}, prometheus=${prometheus}, registry_inmemory=${registry_inmemory}, settings=${settings}, timeout=${timeout}"
+echo "Running with: kind=${kind}, k8s_version=${k8s_version}, olm_version=${olm_version}, olm=${olm}, globalnet=${globalnet}, prometheus=${prometheus}, registry_inmemory=${registry_inmemory}, settings=${settings}, timeout=${timeout}"
```

Comment:
I wouldn't add this here, this printout is for user supplied variables.

The kind binary used will be visible on the debug print anyhow, especially:
```
[14:56:55.391] [shipyard]$ [cluster1] kind-0.14 version
[cluster1] kind v0.14.0 go1.18.2 linux/amd64
```